pyproject.toml

```diff
@@ -100,64 +100,73 @@ test_interfaces = "geoips.commandline.test_interfaces:main"
+smap_remss_winds_netcdf = "geoips.plugins.modules.readers.smap_remss_winds_netcdf"
+smos_winds_netcdf = "geoips.plugins.modules.readers.smos_winds_netcdf"
+scat_knmi_winds_netcdf = "geoips.plugins.modules.readers.scat_knmi_winds_netcdf"
+windsat_remss_winds_netcdf = "geoips.plugins.modules.readers.windsat_remss_winds_netcdf"
+sar_winds_netcdf = "geoips.plugins.modules.readers.sar_winds_netcdf"
+sfc_winds_text = "geoips.plugins.modules.readers.sfc_winds_text"
+ssmi_binary = "geoips.plugins.modules.readers.ssmi_binary"
+ssmis_binary = "geoips.plugins.modules.readers.ssmis_binary"
+viirs_netcdf = "geoips.plugins.modules.readers.viirs_netcdf"
+wfabba_ascii = "geoips.plugins.modules.readers.wfabba_ascii"
+windsat_idr37_binary = "geoips.plugins.modules.readers.windsat_idr37_binary"
 
 [project.entry-points."geoips.output_formatters"]
-full_disk_image = "geoips.plugins.modules.output_formatters.full_disk_image:full_disk_image"
-unprojected_image = "geoips.plugins.modules.output_formatters.unprojected_image:unprojected_image"
-geotiff_standard = "geoips.plugins.modules.output_formatters.geotiff_standard:geotiff_standard"
-imagery_annotated = "geoips.plugins.modules.output_formatters.imagery_annotated:imagery_annotated"
-imagery_clean = "geoips.plugins.modules.output_formatters.imagery_clean:imagery_clean"
-imagery_windbarbs = "geoips.plugins.modules.output_formatters.imagery_windbarbs:imagery_windbarbs"
-imagery_windbarbs_clean = "geoips.plugins.modules.output_formatters.imagery_windbarbs_clean:imagery_windbarbs_clean"
-netcdf_geoips = "geoips.plugins.modules.output_formatters.netcdf_geoips:netcdf_geoips"
-netcdf_xarray = "geoips.plugins.modules.output_formatters.netcdf_xarray:netcdf_xarray"
-text_winds = "geoips.plugins.modules.output_formatters.text_winds:text_winds"
-metadata_default = "geoips.plugins.modules.output_formatters.metadata_default:metadata_default"
-metadata_tc = "geoips.plugins.modules.output_formatters.metadata_tc:metadata_tc"
-
+full_disk_image = "geoips.plugins.modules.output_formatters.full_disk_image"
+unprojected_image = "geoips.plugins.modules.output_formatters.unprojected_image"
+geotiff_standard = "geoips.plugins.modules.output_formatters.geotiff_standard"
+imagery_annotated = "geoips.plugins.modules.output_formatters.imagery_annotated"
+imagery_clean = "geoips.plugins.modules.output_formatters.imagery_clean"
+imagery_windbarbs = "geoips.plugins.modules.output_formatters.imagery_windbarbs"
+imagery_windbarbs_clean = "geoips.plugins.modules.output_formatters.imagery_windbarbs_clean"
+netcdf_geoips = "geoips.plugins.modules.output_formatters.netcdf_geoips"
+netcdf_xarray = "geoips.plugins.modules.output_formatters.netcdf_xarray"
+text_winds = "geoips.plugins.modules.output_formatters.text_winds"
+metadata_default = "geoips.plugins.modules.output_formatters.metadata_default"
+metadata_tc = "geoips.plugins.modules.output_formatters.metadata_tc"
+
+# sed -i "s,pmw_tb.pmw_37pct,pmw_37pct" geoips/plugins/yaml/product*/*
```

Comment:
@jsolbrig I left notes as to search/replace commands to update the YAML plugins. I used these to update the old product_params successfully